master/networking/use-ipvs.md

```diff
@@ -0,0 +1,59 @@
+
+| **kube-proxy mode** | **Designed to be...**                                        | **Linux kernel hooks**                 | **Connection processing overhead...**       |
+| ------------------- | ------------------------------------------------------------ | -------------------------------------- | ------------------------------------------- |
+| iptables            | An efficient firewall                                        | NAT pre-routing using sequential rules | Grows proportional to cluster size          |
+| ipvs                | A load balancer with scheduling options like round-robin, shortest-expected delay, least connections, etc. | Optimized lookup routine               | Stays constant, independent of cluster size |
+
+If you are wondering about the performance differences between iptables and ipvs, the answers are definitely not straightforward. For a comparison between iptables (including {{site.prodname}}’s own use of iptables) and ipvs modes, see [Comparing kube-proxy modes: iptables or IPVS? ](https://www.tigera.io/blog/comparing-kube-proxy-modes-iptables-or-ipvs/).
+
+#### IPVS mode and NodePort ranges
+
+Kube-proxy IPVS mode supports NodePort services and cluster IPs. {{site.prodname}} also uses NodePorts for routing traffic to the cluster, including the same default Kubernetes NodePort range (30000:32767).  If you change your default NodePort range in Kubernetes, you must also change it on {{site.prodname}} to maintain ipvs coverage. 
+
+#### iptables: when to change mark bits
+
+To police traffic in IPVS mode, {{site.prodname}} uses additional iptables mark bits to store an ID for each local {{site.prodname}} endpoint. If you are planning to run more than 1,022 pods per host with IPVS enabled, you may need to adjust the mark bit size using the `IptablesMarkMask` parameter in {{site.prodname}} [FelixConfiguration]({{site.baseurl}}/{{page.version}}/reference/felix/configuration#ipvs-bits). 
+
+#### {{site.prodname}} auto detects ipvs mode
+
+When {{site.prodname}} detects that kube-proxy is running in IPVS mode (during or after installation), IPVS support is automatically activated. 
+
+### Before you begin...
+
+**Required**
+
+- kube-proxy is configured to use IPVS mode
+- Services for ipvs mode are type, NodePort
+
+### How to
+
+As previously discussed, there is nothing you need to do in {{site.prodname}} to use IPVS mode; if enabled, the mode is automatically detected. However, if your default Kubernetes NodePort range changes, use the following instructions to update {{site.prodname}} nodeport ranges to stay in sync. 
+
+#### Change {{site.prodname}} default nodeport range
+
+Using the Felix environment variable, `FELIX_KUBENODEPORTRANGES`, change the default node port range in {{site.prodname}} to match your new default range in Kubernetes. For help, see [Felix environment variables](https://docs.projectcalico.org/master/reference/felix/configuration#ipvs-bits). 
```

Comment:
We should recommend using the `FelixConfiguration` API resource rather than an environment variable here.

Reply:
There is no KubeNodePortRange parameter in the FelixConfiguration API docs. On hold until this PR is fixed by Dev: https://tigera.atlassian.net/browse/OS-5317